eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -429,6 +432,8 @@ data EnactState era = EnactState
   -- ^ Hash of the Constitution
   , ensProtVer :: !ProtVer
   , ensPParams :: !(PParams era)
+  , ensTreasury :: !Coin
```

Comment:
are we moving the tracking of the treasury to here? it's a bit weird to me to have it be separated from the reserves.

Reply:
We're just going to temporarily copy the treasury into `EnactState` and then apply the changes back to the original treasury after the rule is run.

Reply:
That's the plan for now at least

Reply:
That applies to the other fields as well.

Reply:
gotcha! :pray: